ITE/9/README.adoc

```diff
@@ -45,7 +45,29 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification's front matter MUST unambiguously declare a predicate type, a
+version, and a shorter predicate name following the rules defined by the
+attestation framework. The predicate type along with the version maps to the
+`predicateType` field in the `Statement` layer of an in-toto attestation. The
```

Comment:
Minor nit: start a new paragraph after "in-toto attestation."

Reply:
I split this and the example up into a separate paragraph.